src/common/dto/courses/CourseInstanceUpdate.dto.ts

```diff
@@ -0,0 +1,47 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsEnum,
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+} from 'class-validator';
+import { OFFERED } from 'common/constants';
+
+/**
+ * @module Server.DTOS.Courses
+ */
+
+export default abstract class CourseInstanceUpdateDTO {
+  @ApiProperty({
+    type: 'string',
+    example: OFFERED.Y,
+    enum: OFFERED,
+  })
+  @IsEnum(OFFERED)
+  @IsNotEmpty()
```

Comment:
This is actually incompatible with `@IsEnum(OFFERED)`, since one of the valid values is `OFFERED.BLANK`, which is a blank string.

So at present, trying to set a course instance's `offered` field to `OFFERED.BLANK` is returning a 400 validation error, which, incidentally, is breaking the app. I think it's an issue with how validation errors are being handled in the modal component itself...